libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -636,6 +656,13 @@ private async Task<DialogTurnResult> CheckForMultiTurnPromptAsync(WaterfallStepC
             return await stepContext.NextAsync(stepContext.Result, cancellationToken).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Gets the options the dialog will use to display query results to the user.
```

Comment:
Consider improving the quality of this doc comment.

Reply:
Fixed.

---
In reply to: [430550423](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430550423) [](ancestors = 430550423)